makefile

```diff
@@ -1,10 +1,13 @@
 .PHONY: clean build publish docs
 
+install_frontend:
+	pushd . && cd termpair/frontend_src && yarn install && popd
+
 watch_frontend:
-	cd termpair/frontend_src && yarn start
+	pushd . && cd termpair/frontend_src && yarn start && popd
```

Comment:
Is the `pushd`/`popd` required on your machine to end up in the same directory after `make watch_frontend` exits? It isn't on mine. I believe each command in the target is run in its own shell.

Reply:
Hmm, strange. At first it didn't bring me back to the directory where I launched the command from, but now it works fine. I'll remove those `pushd/popd`!